FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -133,8 +133,8 @@ typedef enum
     ReportStatusRejected
 } ReportStatus_t;
 
-/** 
- * @brief Each compilation unit that consumes the NetworkContext must define it. 
+/**
```

Comment:
The comment at the top of this file that describes about the demo needs to be updated to mention about collection and reporting of custom metrics